### ISSUE TITLE
Target request: FLYWOOF722DUAL

### ISSUE THREAD

**Issue by lossUAVes** [iNavFlight/inav#5110]
New FC with baro and 20x20 mouting holes. Very interesting.
![image](https://user-images.githubusercontent.com/24512116/65908151-1d881c80-e3c6-11e9-96f9-b4e4cc33111c.png)
https://flywoo.net/collections/frontpage/products/goku-fc-f722-mini-dual-flight-controller-built-in-osd-5v-9v-2a-bec-icm20689-mpu6000-for-rc-drone-1

**Comment by DzikuVx** [comment]
@lossUAVes please see https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

**Comment by lossUAVes** [comment]
I'm sorry. I was just trying to help.

**Comment by DzikuVx** [comment]
@lossUAVes no need to apologize. I'm just informing what is the current INAV policy towards new targets

**Comment by lossUAVes** [comment]
Thank you. I did not know it. I thought you could migrate from betaflight.

**Comment by echma** [comment]
Hi. I'm sorry to open this again but is there no news? I was just looking through INAV 2.5 and this board is sadly not supported. I only found this build: inav_2.3.0_FLYWOOF7DUAL.hex

**Comment by digitalentity** [comment]
@echma please see https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

TL;DR: we need samples and specs from the manufacturer to be able to add this target. This is done to avoid having a million of scarcely used targets.
